.travis.yml

```diff
@@ -15,7 +15,7 @@ addons:
     sources:
       - avsm
     packages:
-      - opam>=2
+      - opam>=2.0.1
```

Comment:
Why do you need this version ?

Reply:
It's not that specific version, but some builds failed because of the `--disable-sandox` option, and using this lower bound seemed to fix the problem. But we could probably drop this modification and look for a cleaner fix if the problem re-appears.